content/chat/channels.textile

```diff
@@ -0,0 +1,105 @@
+---
+title: Channels and Messages
+meta_description: "Channels bring forward the capability to provide real-time communication between multiple people."
+product: chat
+languages:
+  - javascript
+---
+
+Channels are the core functionality for any chat application or any application with chat features. Channels allow you to separate chat channels and their messages into categories. By using the Publish/Subscribe (Pub/Sub) architectural pattern, your application can have any number of chat participants "join" a channel by joining (subscribing) this channel and receive any messages that these subscribed channels have published. The chat participants are able to "send" messages by publishing them to the specific chat channel that they're subscribed to. The ability to publish and subscribe is completely decoupled from each other allowing the applications to be more flexible and scalable.
+
+Clients can subscribe to channels and publish messages using the realtime interface of an Ably SDK. The REST interface only allows clients to publish messages.
+
+<img src="/images/products/chat/chat.svg" alt="Example of a chat message sent by a user called Claire Jones, with reactions added to the message and a reply from Siti Tengku.">
+
+h2(#namespaces). Namespaces
+
+Group and organise the chat channels by using namespaces, which are based on a prefix that is included as part of the channel name. A colon (:) is used to delimit a channel namespace, where the first segment up until the first colon defines the namespace, any segments after this is defined by you. If a channel name does not contain a colon, the namespace is the entire channel name.
+
+The following are examples of channels that are all part of the "channel" namespace:
+
+* channel:group:name
+* channel:pm:user-id-1:user-id-2
+
+Namespaces can be used to apply operations to all channels within the namespace, such as capabilities, channel rules and integration rules.
+
```

Comment:
Can we add links to channel and integration rules?

Reply:
https://github.com/ably/docs/pull/2090/commits/1be811e27113ef31867310282f6efdad27d55ee5